app/views/groups/_header.html.erb

```diff
@@ -9,10 +9,11 @@
       <li class="nav-item active">
         <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
       </li>
+      <% if user_signed_in? %>
```

Comment:
- [ ] For each category, the user can see their name, icon, and the total amount of all the transactions that belong to that category. You have left with implementing the last one "Total amount of all transactions should be displayed  for each category"

Reply:
- [ ] At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category). You should implement this at the top not at the bottom
- [ ] For a given category, the list of transactions is presented ordered by the most recent, currently you the last one appears at the end at your app

Reply:
- [ ] When I try to add a transaction without filling in the name and amount the app crashes, this should not happen
![Screenshot from 2022-04-23 22-44-44](https://user-images.githubusercontent.com/62705947/164945444-04a60966-2fdc-42cb-8df5-7acaed11d59c.png)

Reply:
- [ ] Please add more tests that cover your app: - App has unit tests for all the most important components of the RoR application and App has integration tests for all the most important components of the RoR application.